backend/error-tags/error-tags.go

```diff
@@ -0,0 +1,8 @@
+package error_tags
```

Comment:
@Vadman97 Is this the right place to define a new model?

Reply:
`MatchedErrorTag` is something defined in `schema.resolvers.graphqls` so you don't need to define it again - just use the generated one from modelInputs.MatchedErrorTag